runtimes/peregrine/src/weights/pallet_democracy.rs

```diff
@@ -390,6 +390,24 @@ impl<T: frame_system::Config> pallet_democracy::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+	fn clear_external_metadata() -> Weight {
+		unimplemented!()
```

Comment:
can we not do this? There was a team that had a stalled chain because they unintentionally pushed code like this to the chain. 😓

Reply:
Oh yeah? Which one was that? I changed that to return `Weight::MAX` so that if we forget to benchmark it for some reason, none of the new txs will be included in a block and the chain won't stall -> https://github.com/KILTprotocol/kilt-node/pull/500/commits/ac590deb38eb0f0f34cde55b586a9d725eb7a331.

Reply:
I would simply run the benchmarks on the compile server for this specific pallet. IMHO that should be fine?

Reply:
Ok will do. I just don't see how is it different to give completely off but benchmarked weights or give it a weight that would prevent this tx from being included. If anything, I think the latter option is safer 😄